### PR TITLE
Plat 641 use mpi mediator

### PR DESCRIPTION
**Pull request by bradsawadye**

### CODE REVIEW

File: swarm.sh [hunk 1]
```diff
@@ -113,15 +100,17 @@ function initialize_package() {
...
+    fi
   ) || {
     log error "Failed to deploy DISI package"
     exit 1
   }
 }
 
 function destroy_package() {
-  docker::service_destroy "${ALL_SERVICES[@]}"
+  docker::service_destroy "${SERVICE_NAMES[@]}"
```
Comment (nour-borgi) [inline swarm.sh:113]: When destroying the config importers should be removed as well, not sure why you would remove it ?

File: .env.local [hunk 2]
```diff
@@ -13,3 +13,13 @@ LOGSTASH_PACKAGE_PATH=
 JS_REPORT_DEV_MOUNT=false
 JS_REPORT_PACKAGE_PATH=
 
+CLUSTERED_MODE=false
+
+MPI_CLIENT_ID=DISI CLIENT
+MPI_CLIENT_SECRET=DISI
+
+# Disables Fhir validation agains IG
```
Comment (nour-borgi) [inline .env.local:21]:
```suggestion
# Disables Fhir validation against IG
```

File: .env.qa [hunk 3]
```diff
@@ -19,3 +19,14 @@ KIBANA_SSL=true
 
 # Data Mapper - Logstash
 LS_JAVA_OPTS=-Xmx2g -Xms2g
+
+CLUSTERED_MODE=false
+
+MPI_CLIENT_ID=DISI CLIENT
+MPI_CLIENT_SECRET=DISI
+
+# Disables Fhir validation agains IG
```
Comment (nour-borgi) [inline .env.qa:28]:
```suggestion
# Disables Fhir validation against IG
```

File: .env.stg [hunk 4]
```diff
@@ -19,3 +19,14 @@ KIBANA_SSL=true
 
 # Data Mapper - Logstash
 LS_JAVA_OPTS=-Xmx2g -Xms2g
+
+CLUSTERED_MODE=false
+
+MPI_CLIENT_ID=DISI CLIENT
+MPI_CLIENT_SECRET=DISI
+
+# Disables Fhir validation agains IG
```
Comment (nour-borgi) [inline .env.stg:28]:
```suggestion
# Disables Fhir validation against IG
```

File: .env.test [hunk 5]
```diff
@@ -19,3 +19,14 @@ KIBANA_SSL=true
 
 # Data Mapper - Logstash
 LS_JAVA_OPTS=-Xmx2g -Xms2g
+
+CLUSTERED_MODE=false
+
+MPI_CLIENT_ID=DISI CLIENT
+MPI_CLIENT_SECRET=DISI
+
+# Disables Fhir validation agains IG
```
Comment (nour-borgi) [inline .env.test:28]:
```suggestion
# Disables Fhir validation against IG
```

**Review by nour-borgi — COMMENTED** [review]
Nice ! Just few comments

File: get-cli.sh [hunk 6]
```diff
@@ -6,30 +6,30 @@ cli_version=${2:-latest}
 case ${option} in
```
[the remark this answers is not in the document]
Comment (michaelloosen) [reply get-cli.sh:6]: No, we created the latest tag that would keep updating to the newest binary released. The platform image version should be pinned in the config.yml file.

File: .env.local [hunk 7]
```diff
@@ -13,3 +13,13 @@ LOGSTASH_PACKAGE_PATH=
...
+DISABLE_VALIDATION=true
+
+# Name of folder containing the Fhir IG importer scripts in the importer folder
+FHIR_CONFIG_IMPORTER_FOLDER_NAME=hapi-fhir
```
Comment (michaelloosen) [inline .env.local:25]: We can remove this from the `.env` var files if we have the default in `package-metadata` anyway.

**Review by michaelloosen — CHANGES_REQUESTED** [review]

File: swarm.sh (hunk 1 above)
Comment (bradsawadye) [reply swarm.sh:113]: The config importers are removed soon after they have been run when we deploy (init/up command). We make use of the `docker::deploy_config_importer` function which removes the importers after they are run.

Comment (michaelloosen) [reply swarm.sh:113]: Yeah, I think we have kept it in the destroy section in the past to ensure that if some weren't removed during the init/up commands, they get removed when we destroy, I think that is what Nour is referring to.

**Review by michaelloosen — APPROVED** [review]

File: swarm.sh [hunk 8]
```diff
@@ -19,32 +18,25 @@ function init_vars() {
...
   )
-  ALL_SERVICES=(
+
+  ALL_SERVICE_NAMES=(
     "${SERVICE_NAMES[@]}"
-    "${SERVICE_IMPORTER_NAMES[@]}"
+    "disi-jsreport-config-importer"
+    "disi-es-index-importer"
+    "disi-kibana-config-importer"
+    "sante-mpi-config-importer"
```
Comment (nour-borgi) [inline swarm.sh:31]:
```suggestion
    "sante-mpi-config-importer"
    "disi-openhim-config-importer"
```

**Review by nour-borgi — APPROVED** [review]

**Pull request merged**